src/App.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from "react";
+import "./App.css";
+
+class App extends Component{
+    render(){
+        return (
+            <main>
+            <header className='navBar'>
```

Comment:
Good attempt, you've got the setup done well. The only things that need to be revised are aesthetic and not functional. 

If you revisit, the mockup you will see that there are 2 color schemes present. Dark color and white. 

Your classes```.navbar``` and ```.main``` are the only ones that should have a dark background color. The sections showing the features have a full-width white background